src/Umbraco.Web.UI.Client/package.json

```diff
@@ -164,8 +165,6 @@
 		"example": "node ./devops/example-runner/index.js",
 		"format:fix": "npm run format -- --write",
 		"format": "prettier 'src/**/*.ts' --check",
-		"generate:server-api-dev": "openapi-ts --file devops/openapi-ts/openapi-ts.dev.config.js",
-		"generate:server-api": "openapi-ts --file devops/openapi-ts/openapi-ts.config.js",
```

Comment:
Not a showstopper, wondering if we could we still have these commands, with paths to the `core` package?

I'm thinking about cognitive load (and muscle memory), as I can see this taking me a few minutes to remember where the `generate:server-api` command now lives.